src/lib/type-policies/proposal.ts

```diff
@@ -9,8 +9,6 @@ export function getProposalName(change: Proposals_proposals_terms_change) {
     return `Update Market: ${change.marketId}`;
   } else if (change.__typename === "UpdateNetworkParameter") {
     return `Update Network: ${change.networkParameter.key}`;
-  } else if (change.__typename === "NewFreeform") {
-    return `Freeform: ${change.hash}`;
```

Comment:
chore (blocking): we will still need to be able to display a title for freeform governance proposals. Suggest taking the whole proposal object here rather than just terms and using rationale for this field to show the name :+1:

Reply:
Do you think we can use `proposal.rational.description` for all titles and remove this logic?

Reply:
i'll let you two decide on that, happy to strip it out and use the description for it, but I'm still a bit out of context regarding the bigger picture :)

Reply:
Descriptions is 1024 characters and is more a summary, I think URL is probably the correct one to use with a link eventuaally